### PR TITLE
Adding AMD SEV-SNP attestation Page

### PR DESCRIPTION
**Pull request by carlosbravoa**

### CODE REVIEW

File: aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst
Comment (rpocase) [inline aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst:37]: should this be `AmdSevSnp`?

File: aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst
Comment (rpocase) [inline aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst:52]: fyi, we'll be able to replace this with snap installation instructions at some point. currently blocked on [upstream support](https://forum.snapcraft.io/t/access-to-dev-cpu-0-msr-from-a-strict-snap/41228) for certain snap confinement options

**Review by rpocase — CHANGES_REQUESTED** [review]

File: aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst
Comment (carlosbravoa) [reply aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst:52]: That is way better. We can update this doc as soon as we have the snap

**Review by rpocase — APPROVED** [review]
lgtm pending CI being happy

**Pull request merged**
